client/lib/react-test-env-setup/index.js

```diff
@@ -45,3 +45,13 @@ module.exports.cleanup = function() {
 	delete global.localStorage;
 	delete global.XMLHttpRequest;
 };
+
+module.exports.auto = function( markup, features ) {
```

Comment:
>needs a better name though. `auto` is weird. any ideas?

I don't find it to be too confusing. Perhaps add or substitute something about lifecycle, setup, teardown, etc.?